### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Keyboard:The first focus is present on the next button.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6798]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS2.4.3; #Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS References:MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.

**Actual:**
On hitting enter from step 9 , the page "select Extract Dacpac Settings" opens and the focus is present on the "Next" button.

**Same issue is observed for:**
1.Export Bacpac Settings:
Follow the exact above steps but select the fourth radio button

**Expected:**
The focus as soon as the page is opened should be present on the first interactive control ie"Source server" drop down.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: [https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the keyboard user needs to change the data that is autocompleted in the fields, he would have to take an extra effort in doing so. As well as the flow of the content will not be know to the user. 

**MAS Reference:**
MAS2.4.3:[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9})

**Attachment for Reference:**
[#6798A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3515087/6798A11y_AzureDataStudio.Schema.Dacpac.Backup._ExtractDacpac_ExtractDacpacSettings_Keyboard.pptx)

**Comment by fsteffi** [comment]
The issue still reproduces in the below version 
Version: 1.12.0-insider
Commit: a584aca96997a18ecedee73a575536332e542718
Date: 2019-09-19T07:58:50.448Z
VS Code 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

[#6798 A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3686165/6798.A11y_AzureDataStudio.Schema.Dacpac.Backup._ExtractDacpac_ExtractDacpacSettings_Keyboard.pptx)

**Comment by udeeshagautam** [comment]
Seems to have regressed again!

**Comment by fsteffi** [comment]
@udeeshagautam yes in-case the user wants to change the auto filled fields, the focus needs to be present at the first interactive element.

**Comment by udeeshagautam** [comment]
@fsteffi thanks for the clarification but I want to mention that the focus can go to fields through Shift + Tab or equivalent if someone explicitly wanted to change but for regular flow they shouldn't need to change. that's why the question on where the default focus should be.

**Comment by fsteffi** [comment]
@udeeshagautam the shift+tab navigation can be usesd but it would require unnecessary efforts that can be avoided by simple navigation. Even if they don't want to change the details , the first focus needs to be on the first dropdown this would help screen reader users navigate through all the information present on screen.

**Comment by udeeshagautam** [comment]
We discussed with Ryan and Alexandria on the workaround that
1. controls can be accessed through shift + tab
2. The page would announce itself when changed (so that the screen reader user is aware that the current focus after clicking the next button on previous page changed)

This is okay workaround to live with for now as per Ryan hence we are clsong the bug. Please let us know if the worakaround doenst work.
